src/index.js

```diff
@@ -1,5 +1,23 @@
 export { mix, is, like }
 
+/**
+ * Accepts an optional superclass as the first argument,
+ * then a bunch of mixins and an optional class factory as the last argument and returns a mixin.
+ * Mostly, you will be using mix with a factory to create mixins, like this:
+ *
+ * var Looker = mix(superclass => class Looker extends superclass {
+ *   constructor() {
+ *     super()
+ *     console.info('A looker is born!')
+ *   }
+ *   look() {
+ *     console.info('Looking good!')
+ *   }
+ * })
+ *
+ * @param {function} args
```

Comment:
I am not sure if it is expressable with JSDoc, but ideally this would read:

@param {function} superclass (optional)
@param {function} mixins...  (0 or more)
@param {function} factory  (optional)

At least one parameter is required.